frontend/webservice/command.cgi

```diff
@@ -143,22 +143,41 @@ sub run_command {
 
     my $config = GRNOC::Config->new(config_file => '/etc/oess/.passwd.xml');
 
-    my $proxy = GRNOC::RouterProxy->new(
-        hostname    => $node,
-        port        => 22,
-        username    => $config->get('/config/@default_user')->[0],
-        password    => $config->get('/config/@default_pw')->[0],
-        method      => 'ssh',
-        type        => 'junos',
-        maxlines    => 1000,
-        timeout     => 15
-    );
+    my $proxy;
+    if ($node->{vendor} eq 'junos') {
+    	$proxy = GRNOC::RouterProxy->new(
+            hostname    => $node,
+            port        => 22,
+            username    => $config->get('/config/@default_user')->[0],
+            password    => $config->get('/config/@default_pw')->[0],
+            method      => 'ssh',
+            type        => 'junos',
+            maxlines    => 1000,
+            timeout     => 15
+    	);
+    } elsif ($node->{vendor} eq 'cisco') {
+	$proxy = GRNOC::RouterProxy->new(
+	    hostname    => $node,
+            port        => 22,
+            username    => $config->get('/config/@default_user')->[0],
+            password    => $config->get('/config/@default_pw')->[0],
+            method      => 'ios2SSH',
```

Comment:
`method` here is what protocol is used to connect to the device's CLI. Only `ssh` and `telnet` are valid values. In this case (basically all cases) we want to use `ssh`.